proxy/prefect_flows_runner.py

```
@@ -66,26 +63,251 @@
     except Exception as error:  # pylint: disable=broad-exception-caught
-        logger.error(str(error))
+        run_logger.error("airbyte connection sync failed: %s", str(error))
         raise
 
+    try:
+        extra = connection_block.extra or {}
+        await _run_post_sync_ops(
+            env=extra.get("env", {}),
+            ops=extra.get("post_sync_ops", []),
+        )
+    except Exception as err:  # pylint: disable=broad-exception-caught
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)